mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/ReprocessingContextInformationDTO.java

```diff
@@ -58,15 +58,27 @@ public String getMailboxId() {
     }
 
     public static class ReprocessingContextInformationForErrorRecoveryIndexationTask extends ReprocessingContextInformation {
-        public static final AdditionalInformationDTOModule<ReprocessingContextInformationForErrorRecoveryIndexationTask, ReprocessingContextInformationDTO> serializationModule(MailboxId.Factory mailboxIdFactory) {
+
+        public static class DTO extends ReprocessingContextInformationDTO {
```

Comment:
do you really want to name this public class DTO?

Reply:
it's an inner class, I think it prevents too long names by adding more context